api/src/controllers/changes.js

```diff
@@ -196,7 +196,8 @@ const getChanges = feed => {
     since: feed.req.query && feed.req.query.since || 0,
     // PouchDB batches requests using this value as a limit and keeps issuing requests until the result
     // set length is lower than the limit.
-    batch_size: MAX_DOC_IDS + 1
+    batch_size: MAX_DOC_IDS + 1,
+    return_docs: false,
```

Comment:
Actually, it's necessary to return docs here, setting it to `false` yields empty results.

Reply:
Ah, we're explicitly using the `complete` event's `results` array?

Reply:
yes.